### DISCUSSION TITLE
🤯 rfmo 2022 开发计划

### DISCUSSION THREAD

**Discussion by Saszr** [Saszr/rfmo#2] [Announcements]
功能性的迭代都会放在 `dev` 分支，差不多再合并 `main` 分支

- [x] 滚动加载
- [x] 分享图片生成
- [x] 富文本编辑器改为 markdown 编辑器，方便插入图片（图片不做本地存储，自行使用外链）
- [ ] 标签功能
- [ ] 侧栏标签固定

**Comment by sseaan** [comment]
催更下标签功能

**Reply by Saszr** [reply]
工作在弄 web3 / rn，排期紧。
过段时间打算直接重构了 web3 账户 + rn + rss 订阅

**Reply by sseaan** [reply]
rss好耶
